### ISSUE TITLE
Getting Cluster from another stack?

### ISSUE THREAD

**Issue by slavafomin** [pulumi/pulumi-awsx#723]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Hello! I have a pretty common setup with two stacks: one stack creates common resources, like clusters and another stack creates application-specific resources. Right now I'm trying to create a `Service` by specifying a `Cluster` from another stack and I can't figure out how to do this. My referenced `Cluster` instance has a type of `<Output<Cluster>>`, while `Service` expects `Cluster`.

I've looked into `Cluster` class and have found a `cluster` constructor property, that could be used to reference existing cluster, but it has a type of `Cluster | pulumi.Input<string>`. I can't figure out how to properly reference the existing cluster from another stack. Is it possible? Thanks!

### Affected area/feature

AWSX: ECS: Service/Cluster

**Comment by slavafomin** [comment]
I've found this example to be useful:
https://www.pulumi.com/docs/guides/crosswalk/aws/ecs/#using-an-existing-ecs-cluster

However, the example is very cumbersome, I would suggest to implement a much simpler way to use an existing cluster.

**Comment by slavafomin** [comment]
> I've found this example to be useful: https://www.pulumi.com/docs/guides/crosswalk/aws/ecs/#using-an-existing-ecs-cluster

Actually, it looks like provided example is not working across stacks. I'm getting the following error:

```
resource 'arn:aws:ecs:eu-central-1:XXXXXXXXXX:cluster/my-cluster' does not exist
```

However, the resource exists in AWS with the same ARN.

**Comment by danielrbradley** [comment]
Hello, a few quick things to check first:

1. When you're calling [`aws.ecs.Cluster.get(...)`](https://www.pulumi.com/registry/packages/aws/api-docs/ecs/cluster/#look-up) are you passing in the name of the cluster rather than the ARN? 
2. What region is the existing cluster based in? Is this the same as the region configured for your default AWS provider?
3. Using the same credentials, do you have access to that cluster using the AWS CLI?

**Comment by danielrbradley** [comment]
This issue is now resolved in the 1.0.0-beta release of the provider. The ECS components have been reworked to break the dependency on using the AWSX cluster component - you can now just provider a cluster ARN.
